client/profile-wizard/steps/business-details/flows/selective-bundle/index.js

```diff
@@ -58,6 +58,26 @@ export const filterBusinessExtensions = ( extensionInstallationOptions ) => {
 	);
 };
 
+export const prepareInstalledExtensionsForTracking = (
```

Comment:
Very nitpicky, but for brevity and since this applies to both installed and not installed extensions:

```suggestion
export const prepareExtensionTrackingData = (
```

Reply:
Sounds good to me. The method was renamed in commit 1dc80b9